### ISSUE TITLE
Demo is not working for f sake

### ISSUE THREAD

**Issue by FlippFlopps** [giotiskl/filterizr#138]
**Describe the bug**
Uncaught TypeError: window.Filterizr is not a constructor

**Filterizr version**
2.2.3 (or 2.2.2)

**To Reproduce**
1. Download zip
2. Run demo (index.html)

**Expected behavior**
For it to work.

**Screenshots**
None

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 81.0.4044.129 (Official Build) (64-bit)

**Additional context**
One would expect for an official demo to work, but nooo.

**Comment by giotiskl** [comment]
@FlippFlopps although your attitude doesn't warrant much help, I will answer this for reference for any other user that might wish to run the demo.

To begin with, the demo directory is not an official demo by any means. It is used solely by me to facilitate development.

That said, double clicking the index.html file won't get the thing to run.

Once you have cloned the repo (or downloaded the .zip):
* Go to the directory where you extracted it
* Run `npm install` to install the project's dependencies
* Run `npm start` to begin the development server which will launch the demo.

Next time "for f's sake" before making a post with bad temper to an open source project, which is completely for free, do a bit more research, ask politely and if the project doesn't serve you, build your own or contribute to improve it.

Have a nice day.

**Comment by FlippFlopps** [comment]
I reserve the right to be grumpy. I also reserve the right to PayPal my respect when I make money off any open source projects. I'm using version 1.2.3, but I needed controlsSelector which was later added. Is there a copy/paste I can do to add controlsSelector into v1.2.3? The latest version doesn't render the last column properly on the same html/css setup used with v1.2.3 and for 7 f sakes I don't have the nerves to go inspecting. You're doing a good job, that's what matters.

**Comment by giotiskl** [comment]
https://github.com/giotiskl/filterizr/blob/master/CHANGELOG.md

`controlsSelector` was added in 1.3.0, I'd try upgrading to that version and see if it works fine or not. Althought for future compatibility I'd consider getting everything to work with the latest version.

V2 onwards is a complete rewrite in TypeScript without dependencies on jQuery and many bugfixes.

**Comment by FlippFlopps** [comment]
Thanks for replying. I tried the latest version but it has a full width bug with the scrollbar on desktop chrome (haven't tested further). When the scrollbar is present, the last column is often mistakenly calculated (pushed to next row).

To reproduce the bug:
1. Have your filter items controlled with media queries to completely fill the width of the screen (aka full-width, I'm using several m.queries from 500px up to 4K)
2. layout: 'sameWidth',
3. Make sure to firstly load a small number of items (so the scrollbar doesn't appear)
4. Make sure that 'all' option will cause the scrollbar to appear (aka enough thumbnails to be higher than the screen height)
5. Click your control to show all => OK => they'll align properly because your calc was finished before the scrollbar appeared
6. Click 'all' again => BUG => the last column will not be calculated properly because of the scrollbar

I'm not a 100% sure it's only related to the scrollbar but for the scrollbar I'm certain. I tried it out with both display block and display flex, on bootstrap 4.
